### DISCUSSION TITLE
Tasmota Display and button

### DISCUSSION THREAD

**Discussion by senojed** [arendst/Tasmota#20679] [Q&A] [answered]
Hello, everyone.
I'm struggling a bit with a problem and would appreciate some advice.

Board Wemoos D1 mini with firmware Tasmota Display. 
Pushbutton (physical) connected to D5(GPIO14) and GND. 
LCD 2004 connected via I2C . 

LCD is working fine and Tasmota shows the button (in web UI) to turn on/off the display. 
But the button (physical) pretends as if it doesn't exist, Tasmota doesn't see it at all.

I tested the same connected button (physical) with a normal Tasmota and relay and it worked fine - switching the relay.

I tried connecting the relay to the Tasmota display with the LCD connected and the pushbutton (physical) still didn't work.
Even though the on/off button (in web UI) for the relay showed up in the Tasmota. 

Anyone have any idea what the problem might be. 
Thanks :)

**Comment by sfromis** [comment]
The Button *input* will never "show up", until you do something with it, by default controlling a power output, like a relay, and/or a display. These are what you get in the web UI, not the input. You use the word "button" so many times that it is not always clear if you mean the physical input, or the web UI elements. With both a relay and a display, I'd expect you to get two times power status/toggle.

**Reply by senojed** [reply]
Thank you for your reply! :)
I edited the original post for clarity :)

So how do I know that the (physical) button is working or why the relay didn't work?
In the configuration they (button and relay) had the same number.

Maybe my question should have been how to connect the button to the LED display lighting...

**Reply by sfromis** [reply]
Same number is not relevant, the default coupling is by sequence. If you have both relay and display, the relay should become the first power channel, and the display then the second - Power1 / Power2. With an input configured as a `Button` gpio, one push should then toggle the relay, and two pushes (in quick succession) the display. Or you can create rules to handle things differently.

If you activate `Weblog 3`, you can then use the console to follow what Tasmota is doing, including detection of button inputs.

**Comment by senojed** [comment, marked as answer]
Now I'm really ashamed...

After all the trials and annoyances here on the forum.
I figured out that I have a badly soldered pin...

Thank you very much for your kindness and sorry!
